homework/homework1/task2/head.py

```diff
@@ -0,0 +1,27 @@
+import sys
+from typing import TextIO
+
+
+def print_head(file_stream: TextIO, number_of_lines: int = 10) -> None:
+    for index, line in enumerate(file_stream, start=1):
+        if index > number_of_lines:
+            break
+        print(line, end="")
+
+
+def main(*files: str) -> None:
+    for file in files:
+        try:
+            with open(file, "r") as f:
```

Comment:
Here and in every other file:

`"r"` is actually the default state, you don't need to write it. If you are just opening a file to read as text you can just write `with open(file) as f`